src/ebpf_syscalls.c

```diff
@@ -700,23 +700,17 @@ print_syscalls_table(FILE *f)
  * dump_syscalls_table -- dump the table of syscalls
  */
 int
-dump_syscalls_table(const char *path)
+dump_syscalls_table(FILE *file)
 {
 	int size = sizeof(struct syscall_descriptor);
-	int ret = 0;
+	int count = SC_TBL_SIZE;
```

Comment:
We are dealing with on-media format, so I think we should use fixed size types (uint32_t here) or verify its size using COMPILE_ERROR_ON. Now I see syscall_descriptor has the same problem...